lib/omniauth/login_dot_gov/authorization.rb

```diff
@@ -28,12 +29,35 @@ def redirect_url_params
         }
       end
 
+      def aal
+        if client.aal.nil?
+          nil
+        elsif VALID_AAL_VALUES.include?(client.aal.to_s)
+          client.aal.to_s
+        else
+          raise "Invalid AAL, choose one of #{VALID_AAL_VALUES}"
+        end
+      end
+
       def acr_values
-        if client.ial == 2
+        values = []
+
+        values << if client.ial == 2
           'http://idmanagement.gov/ns/assurance/loa/3'
         else
           'http://idmanagement.gov/ns/assurance/loa/1'
```

Comment:
the IDP accepts `ial` in these URL values now, WDYT of updating while we're here?

```suggestion
          'http://idmanagement.gov/ns/assurance/ial/2'
        else
          'http://idmanagement.gov/ns/assurance/ial/1'
```

Reply:
we actually debated this and talked about doing it as a separate PR for a clearer commit history, but I defer to @mitchellhenke